### PR TITLE
Editorial recommendation for Test Procedure for Audio Description (SC 1.2.5) 

### PR DESCRIPTION
**Pull request by bruce-usab**
- add case for media using player without AD functionality
- change "audio" to "narration
- provide link to, and example of, "media alternative for text"

This PR is to facilitate a conversation with baseline folks about assessing if narration **adequately** describes important visual content in the media,

### CODE REVIEW

File: _baselines/17SyncMedia.md [hunk 1]
```diff
@@ -97,31 +111,38 @@ Evaluation of captions and audio descriptions to assess its equivalence to synch
...
 
 #### Test Instructions
+
 <ol id="5TI">
-    <li id="5TI-1">Enable audio descriptions through multimedia player functions and play the media. If a separate media file with audio descriptions is provided, test that file.</li>
-    <li id="5TI-2">Check that the audio (with audio descriptions enabled) adequately describes important visual content in the media, including information about actions, characters, scene changes, on-screen text, and other visual content. [SC 1.2.5]</li>
+    <li id="5TI-1">Enable audio descriptions through multimedia player functions and play the media.  If a separate media file with audio descriptions is provided, test that file.  If there is not a separate medial file, and there is not an option to enable audio description, test the default file.</li>
+    <li id="5TI-2">Check that the narration (with audio descriptions enabled) adequately describes important visual content in the media, including information about actions, characters, scene changes, on-screen text, and other visual content. [SC 1.2.5]</li>
```
Comment (kengdoj) [inline _baselines/17SyncMedia.md:129]: Audio includes sounds and speech. Narration is just speech.
 
- Would a train whistle be adequate to describe a train passing?

File: _baselines/17SyncMedia.md [hunk 2]
```diff
@@ -97,31 +111,38 @@ Evaluation of captions and audio descriptions to assess its equivalence to synch
...
+
+<p id="5IC">Pre-recorded synchronized multimedia.  Do not include media that is clearly labeled as a <a href="https://www.w3.org/WAI/WCAG22/Understanding/audio-description-prerecorded#dfn-media-alternative-for-text">media alternative for text</a>.  (An example of media alternative for text is ASL version of content.)</p>
 
 #### Test Instructions
+
 <ol id="5TI">
-    <li id="5TI-1">Enable audio descriptions through multimedia player functions and play the media. If a separate media file with audio descriptions is provided, test that file.</li>
-    <li id="5TI-2">Check that the audio (with audio descriptions enabled) adequately describes important visual content in the media, including information about actions, characters, scene changes, on-screen text, and other visual content. [SC 1.2.5]</li>
+    <li id="5TI-1">Enable audio descriptions through multimedia player functions and play the media.  If a separate media file with audio descriptions is provided, test that file.  If there is not a separate medial file, and there is not an option to enable audio description, test the default file.</li>
```
Comment (kengdoj) [inline _baselines/17SyncMedia.md:128]:
```suggestion
    <li id="5TI-1">Enable audio descriptions through multimedia player functions (if available) and play the media.  If a separate media file with audio descriptions is provided, test that file. </li>
```

**Review by kengdoj — CHANGES_REQUESTED** [review]
See my responses/suggestion.

File: _baselines/17SyncMedia.md (hunk 1 above)
Comment (cliffbert) [reply _baselines/17SyncMedia.md:129]: Agreed

File: _baselines/17SyncMedia.md (hunk 2 above)
Comment (cliffbert) [reply _baselines/17SyncMedia.md:128]: Agreed

**Review by kengdoj — CHANGES_REQUESTED** [review]
The PR's suggested change to "narration" is not accepted.

File: _baselines/17SyncMedia.md (hunk 2 above)
Comment (KMSOC) [reply _baselines/17SyncMedia.md:128]: Did you mean to say "medial" file in line 128, or should it be "media" file?

Comment (cliffbert) [reply _baselines/17SyncMedia.md:128]: media

File: _baselines/17SyncMedia.md (hunk 1 above)
Comment (kengdoj) [reply _baselines/17SyncMedia.md:129]: @cliffbert, what do you agree with?

**Pull request closed without merging**